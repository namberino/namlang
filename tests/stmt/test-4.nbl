var a = 3;
var b = 4;
print(a + b);

print("Testing variable printing");
var d = 0;
print(d = 5);
