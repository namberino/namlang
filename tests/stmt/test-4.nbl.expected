Executing file: tests/stmt/test-4.nbl
7.000000
Testing variable printing
5.000000
